backend/src/main/resources/db/migration/V7__review_table_change.sql

```diff
@@ -0,0 +1,9 @@
+-- 이전에 review 테이블 변경 사항 중 new_review가 아닌 review에 적용된 사항을 다시 반영합니다.
+ALTER TABLE new_review ADD COLUMN member_id BIGINT NULL;
+
+-- 기존 review 테이블을 삭제하고, ew_review 테이블의 이름을 "review"로 변경합니다.
+-- 운영 환경에서의 오류 발생 시, 작업 롤백을 위해 트랜잭션 실행합니다.
+BEGIN
+DROP TABLE IF EXISTS review;
+ALTER TABLE new_review RENAME TO review;
+COMMIT
```

Comment:
```suggestion
BEGIN;
  DROP TABLE IF EXISTS review;
  ALTER TABLE new_review RENAME TO review;
COMMIT;
```

BEGIN 과 COMMIT 에도 세미콜론 붙여주세요!

Reply:
![image](https://github.com/user-attachments/assets/ff57d6e5-2409-40fe-91ad-73b377cb988b)

Reply:
호오.. 반영했습니다